qutip/steadystate.py

```diff
@@ -914,6 +914,92 @@ def _iter_count(r):
         return rhoss
 
 
+def steadystate_floquet(H_0, c_ops, Op_t, w_d=1.0, nmax=3, sparse=False):
+    """
+    Calculates the effective steady state for a driven
+     system with a time-dependent cosinusoidal term:
+
+    .. math::
+
+        \\mathcal{\\hat{H}}(t) = \\hat{H}_0 +
+         \\mathcal{\\hat{O}} \\cos(\\omega_d t)
+
+    Parameters
+    ----------
+    H_0 : :obj:`~Qobj`
+        A Hamiltonian or Liouvillian operator.
+
+    c_ops : list
+        A list of collapse operators.
+
+    Op_t : :obj:`~Qobj`
+        The the interaction operator which is multiplied by the cosine
+
+    w_d : float, default 1.0
+        The frequency of the drive
+
+    nmax : int, default 3
```

Comment:
This is the number of iteration, not the maximum.
Why is the default 3? How to know how many is enough?

Reply:
You are right, this is the number of iterations. When the steadystate does not change anymore increasing this number it reaches the convergence. I tried it for different systems and n = 3 seems to be a valid choice to balance resolution and speed.